.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+    "presets": [
+        "@babel/preset-env",
+        "@babel/preset-react"
+    ],
+    "plugins": [
+        [
+            "@babel/plugin-transform-react-jsx",
+            {
+                "runtime": "automatic"
+            }
+        ]
+    ]
+}
```

Comment:
do we need a babelrc file? webui is using the latest storybook and not configuring babel at all: https://github.com/ipfs/ipfs-webui.

Reply:
i do not think it is needed, leftover from previous attempt not using aegir